internal/appconfig/machines.go

```diff
@@ -38,66 +39,83 @@ func (c *Config) ToReleaseMachineConfig() (*api.MachineConfig, error) {
 		Env: lo.Assign(c.Env),
 	}
 
-	machineConfig.Env["RELEASE_COMMAND"] = "1"
+	mConfig.Env["RELEASE_COMMAND"] = "1"
 	if c.PrimaryRegion != "" {
-		machineConfig.Env["PRIMARY_REGION"] = c.PrimaryRegion
+		mConfig.Env["PRIMARY_REGION"] = c.PrimaryRegion
 	}
 
-	return machineConfig, nil
+	return mConfig, nil
 }
 
-func (c *Config) defaultMachineConfig() (*api.MachineConfig, error) {
+//
```

Comment:
just suggesting some sort of comment because having a blank `//` bothers me lol
```suggestion
// updateMachineConfig applies configuration options from the optional MachineConfig passed in, then the base config, into a new MachineConfig
```